src/status_im/events.cljs

```diff
@@ -1291,3 +1294,25 @@
                 (reset-current-profile-chat % (get-in % [:db :contacts/identity]))
 
                 nil))))
+
+(defn on-ramp<-rpc [on-ramp]
+  (clojure.set/rename-keys on-ramp {:logoUrl :logo-url
+                                    :siteUrl :site-url}))
+
+(handlers/register-handler-fx
```

Comment:
events.cljs and register-handler-fx are legacy, we use model.cljs and fx/defn instead